Library/Homebrew/utils/formatter.rb

```diff
@@ -108,4 +108,11 @@ def pluralize(count, singular, plural = nil, show_count: true)
 
     show_count ? "#{count} #{words}" : words
   end
+
+  def comma_and(*items)
+    *items, last = items.map(&:to_s) # rubocop:disable Lint/ShadowedArgument, TODO: Remove when RuboCop 0.57.3 is released.
```

Comment:
Is there an upstream bug you can link to here?

Reply:
Done.